cognite/client/data_classes/extractionpipelines.py

```diff
@@ -203,6 +218,10 @@ def _load(cls, resource: dict, cognite_client: CogniteClient | None = None) -> E
                 ExtractionPipelineContact._load(contact) if isinstance(contact, dict) else contact
                 for contact in instance.contacts
             ]
+        if instance.notification_config and isinstance(instance.notification_config, dict):
```

Comment:
Can we skip the `isinstance` check on `dict`?
```suggestion
        if instance.notification_config:
```

Reply:
No, then mypy complains. I don't like this pattern for loading, but did not what to do more changes than necessary.